muspinsim/tests/test_structure.py

```diff
@@ -327,6 +327,11 @@ def test_compute_closest(self):
             ),
         )
 
+    def test_compute_closest_ignore_all(self):
+        structure = MuonatedStructure(StringIO(TEST_CELL_FILE_DATA), fmt="castep-cell")
+        with self.assertRaises(RuntimeError):
+            structure.compute_closest(1, ignored_symbols=["V", "Si", "H"])
+
```

Comment:
Assuming we end up doing things like I suggested in the other comment, would need:
- Test that we can't ignore the muon symbol (i.e. this test, and maybe one with a custom value for the muon - just as a sanity check)
- Test that if we ignore everything but the muon, this error gets raised (i.e. this test but without the `"H"`)